src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -160,6 +160,9 @@ protected static bool IdentifiersMatch(ISyntaxFactsService syntaxFacts, string n
             FindReferencesDocumentState state,
             CancellationToken cancellationToken)
         {
+            if (!symbol.CanBeReferencedByName())
```

Comment:
Isn't this a property on `ISymbol`?

```suggestion
            if (!symbol.CanBeReferencedByName)
```

Reply:
Yep, amended the commit to correct this.